src/search.cpp

```diff
@@ -967,11 +967,11 @@ Value Search::Worker::search(
                 if (!givesCheck && lmrDepth < 7 && !ss->inCheck)
                 {
                     Piece capturedPiece = pos.piece_on(move.to_sq());
-                    int   futilityEval =
-                      ss->staticEval + 297 + 284 * lmrDepth + PieceValue[capturedPiece]
+                    Value futilityValue =
+                      ss->staticEval + 298 + 284 * lmrDepth + PieceValue[capturedPiece]
```

Comment:
why change the value from 297 to 298?

Reply:
look at `if (futilityValue < alpha)` was changed to `if (futilityValue <= alpha)`
it's the same functionally if futilityValue was only used here which it is.